bin/add_celltypes_to_sce.R

```diff
@@ -42,6 +42,12 @@ option_list <- list(
     type = "character",
     help = "Name of marker by cell type reference file used with CellAssign.
             File name is expected to be in form: `<ref_name>_<source>_<version>.tsv`"
+  ),
+  make_option(
+    opt_str = c("--celltype_ref_metafile"),
+    type = "character",
+    help = "Metadata TSV file containing cell type reference metadata.
+            This file is used to obtain a list of organs used for CellAssign annotation."
```

Comment:
```suggestion
            This file is used to obtain a list of organs used for CellAssign annotation and is only required if CellAssign results provided as input."
```
Let's just be explicit that it's only required if CellAssign results are provided.